test/test_mqtt_base.py

```
@@ -48,13 +56,16 @@ def tearDown(self):
             self.broker_thread.join(timeout=5)
 
             if self.broker_thread.is_alive():
-                logging.info(
+                logging.warning(
```

Throw an exception instead of warning please